plugins/kubernetes/app/controllers/admin/kubernetes/clusters_controller.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 class Admin::Kubernetes::ClustersController < ApplicationController
   before_action :authorize_admin!
-  before_action :authorize_super_admin!, only: [:create, :new, :update, :edit]
+  before_action :authorize_super_admin!, except: [:index, :show, :seed_ecr]
```

Comment:
it does not have negative side-effects, so should be fine for almost anyone to do